LottieSample/src/main/kotlin/com/airbnb/lottie/samples/AppIntroActivity.kt

```diff
@@ -13,29 +13,29 @@ import com.matthewtamlin.sliding_intro_screen_library.buttons.IntroButton
 import com.matthewtamlin.sliding_intro_screen_library.core.IntroActivity
 import com.matthewtamlin.sliding_intro_screen_library.core.LockableViewPager
 
+private val ANIMATION_TIMES = floatArrayOf(0f, 0.3333f, 0.6666f, 1f, 1f)
 
 class AppIntroActivity : IntroActivity() {
-    private val ANIMATION_TIMES = floatArrayOf(0f, 0.3333f, 0.6666f, 1f, 1f)
```

Comment:
@leleliu008 You have to make this `const` then

Reply:
@gpeal I'm sorry for not having explained it clearly. In fact, two points involved here:
1、move to outside of class define as top level object.it will follows the kotlin naming convention.
reference: http://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

2、here we can not use const keyword, because the const keyword can only be used with a value of type String or a primitive type.
reference: http://kotlinlang.org/docs/reference/properties.html#compile-time-constants